MobPlugin/src/main/java/eu/cloudservice/cloudmob/listener/v18_112/ArmorStandListener.java

```diff
@@ -0,0 +1,33 @@
+package eu.cloudservice.cloudmob.listener.v18_112;
+
+import eu.cloudservice.cloudmob.Mob;
+import eu.cloudservice.cloudmob.MobSelector;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.ItemDespawnEvent;
+import org.bukkit.event.player.PlayerArmorStandManipulateEvent;
+
+public final class ArmorStandListener implements Listener {
+
+    @EventHandler
+    public void handle(PlayerArmorStandManipulateEvent e) {
+        Mob mob = MobSelector.getInstance().getMobs().get(e.getRightClicked().getUniqueId());
+        if (mob != null) {
+            e.setCancelled(true);
+        }
+    }
+
+    @EventHandler
+    public void handle(ItemDespawnEvent e) {
```

Comment:
How come the item would despawn?

Reply:
Idk, o have only copied this code.

Reply:
This was a question, not a command to remove this code.
In practice, the item should be spawned with the means necessary for it to never despawn